cpp/CMakeLists.txt

```diff
@@ -82,14 +82,14 @@ if (${USE_ALTERNATE_FORMATS} STREQUAL "ON")
 endif ()
 
 # Find all the required libraries and programs.
-if (${USE_BOOST} STREQUAL "ON")
+if (${USE_BOOST} STREQUAL "ON" OR ${BUILD_GEOCODER} STREQUAL "ON")
```

Comment:
I take this back. In this context this is doing the right thing. My initial reading was wrong.